exetera/core/dataframe.py

```diff
@@ -0,0 +1,245 @@
+from exetera.core import fields as fld
```

Comment:
Can you put the license boiler plate at the top of this file and other new files?